modules/mono/SCsub

```diff
@@ -140,7 +146,7 @@ def mono_build_solution(source, target, env):
         msbuild_path = msbuild_info[0]
         framework_path_override = msbuild_info[1]
     else:
-        msbuild_path = find_msbuild_unix('msbuild')
+        msbuild_path = find_msbuild_unix('msbuild','msbuild.exe')
```

Comment:
I would go with one of the following options here:
- Do the same as the C++ version, just append `.exe` to the first parameter.
- Pass various names, but as an array, to avoid code duplication as seen above.